### PR TITLE
Update python.md

### PR DESCRIPTION
**Pull request by inirudebwoy**
## About the changes
Updating documentation to match what I have done here https://github.com/Unleash/unleash-client-python/pull/207

### CODE REVIEW

**Comment by thomasheartman** [comment]
Hey! Thanks for the PR 😄 I see that this is the same as in the file you linked, but I'm not sure I understand why. I'm probably missing some context, but why are you duplicating it? Using the `fallback_function` argument is already shown just a couple lines down.

**Comment by inirudebwoy** [comment]
> Hey! Thanks for the PR 😄 I see that this is the same as in the file you linked, but I'm not sure I understand why. I'm probably missing some context, but why are you duplicating it? Using the `fallback_function` argument is already shown just a couple lines down.

Hey @thomasheartman 
Those docs have same issue as ones I have fixed. `default_value` keyword argument will raise exception. Maybe the docs need a bit more than that. As you correctly pointed out `fallback_function` is couple lines below my change. The examples show different features though. They show "how to set default" and "how to add fallback function".

**Comment by thomasheartman** [comment]
Thanks for the clarification, but I'm still not sure I understand completely. I'd be very happy to have this change in the docs too, but I just want to make sure I understand what's happening first, so that we can explain it to the users. And if the `default_value` keyword has stopped working, then that's something we should also clarify. When did that happen? 

> Those docs have same issue as ones I have fixed. default_value keyword argument will raise exception. 

Does it? That sounds mighty strange. I couldn't find an issue that describes it. Was there an issue raised? In that case: could you link me to it, please? ☺️ 

> The examples show different features though. They show "how to set default" and "how to add fallback function".

Sorry, I'm probably missing something, but the two snippets appear to be _exactly_ the same. How are they showing two different features?

**Comment by sighphyre** [comment]
@thomasheartman Thanks for finding that PR, I was just looking for that! From what I can tell, yes, that's the source of the default value being removed. The exception being raised here when the default value parameter is passed is just how Python handles bad parameters in a function call

File: website/docs/sdks/python.md [hunk 1]
```diff
@@ -29,7 +29,10 @@ client.is_enabled("My Toggle")
 Specifying a default value:
 
 ```Python
-client.is_enabled("My Toggle", default_value=True)
+def custom_fallback(feature_name: str, context: dict) -> bool:
+    return True
+         
+client.is_enabled("My Toggle", fallback_function=custom_fallback)
```
Comment (sighphyre) [inline website/docs/sdks/python.md:35]: This isn't wrong but a lambda would point out that this is different from the previous call, as per @thomasheartman's comments and also be a bunch simpler which is always nice. Something like 

```unleash_client.is_enabled("ThisFlagDoesn'tExist", fallback_function=lambda: True)```

What do you think?

**Review by sighphyre — COMMENTED** [review]
I agree with this in principle, I've provided a small suggestion on the change. It would also be nice if we could update the docs strings for this method in ./UnleashClient/__init__.py because that also references the old default value. I don't mind doing that in a separate PR though

**Comment by thomasheartman** [comment]
@sighphyre Thanks for jumping in here! Just thought I'd put forth that I added similar changes to the usage.rst file a while back in https://github.com/Unleash/unleash-client-python/pull/208. We could potentially just copy that if you think that's alright?

**Comment by thomasheartman** [comment]
@inirudebwoy Thanks again for making us aware of this and submitting a suggested fix 🙌🏼  I've made changes to the documentation in #1752 which should cover the cases you've described. I'm closing this PR in favor of the other one due to inactivity. If you feel that it doesn't address the issues you were having, please let us know and we'll rework the docs some more ☺️

**Pull request closed without merging**